app/configurator/components/layout.tsx

```diff
@@ -47,7 +47,7 @@ const useStyles = makeStyles<Theme>((theme) => ({
   LMRPanelHeaderLayout: {
     width: "100%",
     display: "grid",
-    gridTemplateColumns: `${DRAWER_WIDTH}px minmax(22rem, 1fr) ${DRAWER_WIDTH}px`,
+    gridTemplateColumns: `${DRAWER_WIDTH}px minmax(22rem, 1fr) auto`,
```

Comment:
I think the idea here was to have a symmetrical layout, not sure if we need to have this change – if you think so, then you should also update `LMRPanelHeaderLayout` class to mimic this 😄 Btw, as it's not used anywhere, maybe we can just remove this layout option :)

Reply:
The problem here was that the publish options + save draft would be wider.